src/main/java/org/isf/sms/providers/common/CustomCommonDecoder.java

```diff
@@ -0,0 +1,32 @@
+package org.isf.sms.providers.common;
```

Comment:
Please include the standard license header to all new files.   Thanks.

Reply:
I corrected as you suggested.